.github/workflows/ci.yml

```diff
@@ -47,9 +47,21 @@ jobs:
         run: yarn lint
 
       - name: Run happo
+        if: github.event_name == 'pull_request'
         run: yarn run happo-ci-github-actions
         working-directory: './packages/visx-demo/'
         env:
           HAPPO_API_KEY: ${{ secrets.HAPPO_API_KEY }}
           HAPPO_API_SECRET: ${{ secrets.HAPPO_API_SECRET }}
           HAPPO_COMMAND: '../../node_modules/happo.io/build/cli.js'
+
+      - name: Report package sizes
+        if: github.event_name == 'pull_request'
+        run: yarn build:sizes && yarn check:sizes
+
+      - name: Deploy gallery
+        if:
```

Comment:
It seems there's no simple "PR merge action" and this is [one recommended check](https://github.community/t/trigger-workflow-only-on-pull-request-merge/17359/6).

We could create a new workflow for this step but it relies on the `yarn install` and `yarn build` steps so they'd need to be repeated. I'm not 100% sure this will work, it seems like if it does that the entire job pipeline runs for PR open, PR close, and PR merge?